templates/init-job.yaml

```diff
@@ -104,5 +104,7 @@ spec:
           resources:
             {{- . | toYaml | nindent 12 }}
           {{- end }}
+          securityContext:
+            {{- toYaml .Values.securityContext | nindent 12 }}
```

Comment:
```suggestion
            {{- toYaml .Values.init.securityContext | nindent 12 }}
```

Reply:
Could we scope the init job's security context inside init? In case the security context of the deployment and init job are different

Reply:
Sounds good. Let me make the changes.